packages/env-tests/src/tests/granda-mento.ts

```diff
@@ -0,0 +1,146 @@
+import { sleep } from '@celo/base'
+import { StableToken } from '@celo/contractkit'
+import { describe, test } from '@jest/globals'
+import BigNumber from 'bignumber.js'
+import { EnvTestContext } from '../context'
+import {
+  fundAccountWithCELO,
+  fundAccountWithStableToken,
+  getKey,
+  ONE,
+  TestAccounts,
+} from '../scaffold'
+
+export function runGrandaMentoTest(context: EnvTestContext, stableTokensToTest: StableToken[]) {
+  const celoAmountToFund = ONE.times(61000)
```

Comment:
I've been going under the assumption this would be tested on staging/alfajores/baklava when we test the governance proposal to activate GrandaMento prior to mainnet activation. Testing on mainnet doesn't feel super practical because the minimum exchange quantities will be pretty large (eg buying/selling 50k cUSD will probably be the min amount). Plus the `env-tests` make strong assumptions about funds being accessible in sorta-insecure keys that are derived from the mnemonic, which probably don't have real funds on mainnet

Reply:
Confirmed this will not be ran on mainnet